docs/DeveloperGuide.adoc

```diff
@@ -456,21 +456,21 @@ since `**Budget**` is not really related to `**Transaction**` in the real world.
 // tag::taguntag[]
 === Tag/Untag feature
 
-We allow the user to add and remove custom `**Tag**` objects in `**Transaction**` entries so that they can categorise and partition the entries to their own liking.
+We allow the user to add and remove custom `**Tag**` objects in `**Transaction**` entries so that they can categorise the entries to their own liking.
 The following sections describe how this feature is implemented and the design considerations that led up to the solution.
 
 ==== Implementation
 
-The **tag**/**untag** command performs modifications on existing `**Transaction**` entries. Due to polymorphism, both `**Expense**` and `**Income**` objects extends the abstract class `**Transaction**`, and are treated the same way in the context of the **tag**/**untag** command.
+The **tag**/**untag** command performs modifications on existing `**Transaction**` entries which reside in the `**Model**`. Implemented using the concept of polymorphism , both `**Expense**` and `**Income**` objects extend the abstract class `**Transaction**`, and are treated the same way in the context of the **tag**/**untag** command.
```

Comment:
entries **that reside** in the `**Model**`

Reply:
Fixed.